### PR TITLE
EZP-30503: Changed names of base layout parameters

### PR DESCRIPTION
**Pull request by pawbuj**
jira: https://jira.ez.no/browse/EZP-30503
related to : https://github.com/ezsystems/ezplatform-admin-ui/pull/986

### CODE REVIEW

**Comment by emodric** [comment]
Renaming `pagelayout` to `page_layout` is quite a big BC break, lots of external templates might depend on it. Since it's already lowercased I'd like to see it kept as it is, to reduce potential issues.

**Comment by lserwatka** [comment]
@pawbuj we should leave out `pagelayout` name, here and in the referenced PRs.

File: eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Common.php [hunk 1]
```diff
@@ -104,6 +104,11 @@ public function addSemanticConfig(NodeBuilder $nodeBuilder)
             ->scalarNode('pagelayout')
                 ->info('The default layout to use')
                 ->example('AppBundle::pagelayout.html.twig')
+                ->setDeprecated('The "pagelayout" option is deprecated. Use "page_layout" instead.')
+            ->end()
+            ->scalarNode('page_layout')
+                ->info('The default layout to use')
+                ->example('AppBundle::pagelayout.html.twig')
```
Comment (alongosz) [inline eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Common.php]: Nitpick: let's also follow the same file naming convention in the example.

File: eZ/Publish/Core/MVC/Symfony/View/ParametersInjector/ViewbaseLayout.php [hunk 2]
```diff
@@ -16,31 +17,37 @@ class ViewbaseLayout implements EventSubscriberInterface
     /**
      * @var string
      */
-    private $pageLayout;
+    private $viewbaseLayout;
 
     /**
-     * @var string
+     * @var ConfigResolverInterface
```
Comment (alongosz) [inline eZ/Publish/Core/MVC/Symfony/View/ParametersInjector/ViewbaseLayout.php]: Nitpick: please use FQCN for the consistency's sake.

File: eZ/Publish/Core/MVC/Symfony/View/ParametersInjector/ViewbaseLayout.php [hunk 3]
```diff
@@ -16,31 +17,37 @@ class ViewbaseLayout implements EventSubscriberInterface
...
-    }
 
     public static function getSubscribedEvents()
     {
         return [ViewEvents::FILTER_VIEW_PARAMETERS => 'injectViewbaseLayout'];
     }
 
+    private function getPageLayout(): string
+    {
+        return $this->configResolver->hasParameter('page_layout') ? $this->configResolver->getParameter('page_layout') : $this->configResolver->getParameter('pagelayout');
```
Comment (alongosz) [inline eZ/Publish/Core/MVC/Symfony/View/ParametersInjector/ViewbaseLayout.php]: Nitpick: readability: I usually try to split long lines with ternary operators:
```php
$this->configResolver->hasParameter('page_layout') 
    ? $this->configResolver->getParameter('page_layout') 
    : $this->configResolver->getParameter('pagelayout');
```

**Pull request merged**